### PR TITLE
yarn upgrade && yarn format-all

### PR DESCRIPTION
**Pull request by casperdcl**
Using:

- current `master` (7139bde)
- node v14.17.4
- `yarn && npm install` as recommended in https://dvc.org/doc/user-guide/contributing/docs#development-environment

Then:

- `yarn format-all`: 10 files changed, 57 insertions(+), 60 deletions(-)
- `npm update`: updates 10 package versions in `package.json`
- `yarn upgrade`: 5775 insertions(+), 5160 deletions(-) in `yarn.lock`

Related:

- `yarn lint-ts`: finds 5 errors (in CI as well as locally) - for follow-up issue?
- #2774

### CODE REVIEW

**Comment by rogermparent** [comment]
outdated `yarn.lock` broke the deploy, just pushed an update

**Comment by casperdcl** [comment]
Ah so do we need to update the instructions over at https://dvc.org/doc/user-guide/contributing/docs#development-environment to remove mentions of `npm` and instead use `yarn`?

**Comment by rogermparent** [comment]
Other than this one entry:

> You may need to resolve dependencies at this point by running:
> 
> `npm install`

It seems that we already do use `yarn` everywhere on that page. The first mention of `npm install -g yarn` is just to get yarn on the system, since if a user doesn't have `yarn` they'll almost certainly have `npm`. That second one should be changed, though, since after `yarn` is installed the user shouldn't be running `npm`.

**Comment by rogermparent** [comment]
#2785 is up for this, thanks for pointing it out!

**Comment by casperdcl** [comment]
ok so can this PR be merged @rogermparent? Or would you like to fix the `lint-ts` stuff first?

**Comment by shcheklein** [comment]
I think we need to fix the issue, CI should be clean to my mind.

**Comment by julieg18** [comment]
> I think we need to fix the issue, CI should be clean to my mind.

@rogermparent, I can help clear the errors if needed!

**Comment by rogermparent** [comment]
> I can help clear the errors if needed!

@julieg18 That'd be great, thanks! Feel free to ping me if you get stuck on something.

**Review by casperdcl — COMMENTED** [review]
lgtm but I can't approve a PR I initiated

**Comment by julieg18** [comment]
@rogermparent, can you review the lint fixes?

File: src/components/Blog/Feed/Item/index.tsx
Comment (rogermparent) [inline src/components/Blog/Feed/Item/index.tsx:73]: If I understand correctly, I think this should be:

```jsx
{image ? (
  <Image fluid={image} className={styles.picture} />
) : (
  <Placeholder className={styles.picture}
)}
```

because we should show a placeholder if there's no `image`, and with this we'll just show nothing if there's `picture` but no `image`- that said, I'm not sure this could actually happen since `image` and `picture` are practically the same thing, so it's more of a stylistic nitpick.

**Review by rogermparent — APPROVED** [review]
One nitpick which looked like it could be a logic problem, but looking at the whole file seems to be nothing but an opportunity for slightly reduced complexity to get the same practical effect.

Everything else looks fine, nice job!

**Comment by casperdcl** [comment]
`yarn lint-ts` still yields 5 errors for me :thinking:

**Comment by casperdcl** [comment]
maybe now after another `yarn upgrade`?

**Comment by julieg18** [comment]
Working now! @casperdcl, is it working for you? I think we're good to merge if so!

**Comment by jorgeorpinel** [comment]
As this was rolled back, should we re-try in 2 separate PRs? One for `yarn update` and another (nested?) one for `yarn  format-all`

**Pull request merged**
